### DISCUSSION TITLE
Cosmote.gr geo-blocked

### DISCUSSION THREAD

**Discussion by nt74** [iptv-org/community#1064] [Q&A]
Just curious about Cosmote.gr EPG and how comes this is geoblocked.
When I use Webgrabplus it isn't and I am also able to see the [TV-guide](https://www.cosmotetv.gr/portal/residential/program) from just browsing.

Are You able to unblock the geoblock?

**Comment by freearhey** [comment]
In general, the guide seems to download normally regardless of location, but if you have an error please report it through this [form](https://github.com/iptv-org/epg/issues/new?assignees=&labels=broken+guide&projects=&template=---broken-guide.yml).
